### PR TITLE
add error logging to healtcheck

### PR DESCRIPTION
**Pull request by diegocepedaw**
if healthcheck fails log the reason why

### CODE REVIEW

File: src/oncall/healthcheck.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 from falcon import HTTPNotFound, HTTPInternalServerError
 from . import db
 
+import logging
+logger = logging.getLogger('oncall.app')
```
Comment (jwon) [inline src/oncall/healthcheck.py:8]: I think this should be `logger = logging.getLogger('oncall.healthcheck')` right?

Comment (jrgp) [reply src/oncall/healthcheck.py:8]: a shorthand for the above ^  would be `logging.getLogger(__name__)` but maybe the logger named "oncall.app" is handled in a specific way?

File: src/oncall/healthcheck.py [hunk 2]
```diff
@@ -29,11 +32,13 @@ def on_get(self, req, resp):
                 cursor.close()
                 connection.close()
             except:
```
Comment (jrgp) [inline src/oncall/healthcheck.py:34]: While we're here, can we replace these bare excepts with `except Exception`, and in the case below `except IOError` ? (Bare excepts can have some unintentional behavior like silencing sys.exit() )

**Review by jrgp — CHANGES_REQUESTED** [review]

File: src/oncall/healthcheck.py (hunk 2 above)
Comment (diegocepedaw) [reply src/oncall/healthcheck.py]: changed

**Review by jwon — APPROVED** [review]

**Pull request merged**